package.json

```diff
@@ -838,6 +838,7 @@
     "antlr4ts": "^0.5.0-alpha.3",
     "archiver": "^5.3.1",
     "async": "^3.2.3",
+    "aws4": "^1.12.0",
```

Comment:
need to add `aws4` module in order to sign requests to new AWS Bedrock connector. https://www.npmjs.com/package/aws4